README.md

```diff
@@ -51,12 +51,33 @@
   <a href="https://tanka.dev/tutorial/overview"><strong>Let's kill some YAML together&nbsp;&nbsp;▶</strong></a>
 </p>
 
+Another Grafana Tanka community call is coming up!
+
+- :calendar: Mark the date: 2020-10-06 16:00 UTC
+- :tv: Join the meet in this doc: https://bit.ly/3czxbFz
```

Comment:
Not outside shorteners, please. Use hrefs.

Reply:
This section shouldn't be here in the first place, forgot to remove it